front-end/src/components/Dashboard/index.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import styled from 'styled-components';
+import SideBarNav from "../../shared/SideBarNav";
+import jwt_decode from 'jwt-decode'
+
+
+
+
+class Dashboard extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            created_at: '',
+            id: '',
+            email: '',
+            password: '',
+            isLoaded: false
+        };
+    }
+        componentDidMount() {
+            let token =  localStorage.getItem('access_token');
+            let decoded = jwt_decode(token);
+            console.log({decoded});
+            return fetch("http://localhost:3000/users/" + decoded.id, {
```

Comment:
Curious to know why you ended up going with `fetch` instead of using `axios`. I just want to make sure we're consistent across the board on the front end.

Reply:
I was using fetch because that was what one of the examples I found was using. I will convert to axios.